website/site/layouts/index.html

```diff
@@ -36,7 +36,7 @@ <h3>{{ .feature | markdownify }}</h3>
       <ol>
         {{ range .Site.Data.updates }}
           {{ range first 3 . }}
-            <a href="{{ .url }}"><li><div class="update-metadata"><span class="update-version">{{ .version }}</span><span class="update-date">{{ dateFormat "January 2, 2006" .date }}</span></div><span class="update-description">{{ .description  | markdownify }}</span></li></a>
```

Comment:
Why not just use the `url` from the updates.yml?

Reply:
That's what this is - `Site.Data.updates` is read in from the yaml.

Reply:
It was changed from `{{ .url }}` to `https://github.com/netlify/netlify-cms/releases/tag/{{ .version }}`. We still have the URL in the `Site.Data.updates`, though, why not just use that url?

Reply:
Oh, sorry, should have looked closer.

We're going to drop the urls entirely - that way we don't have to manually enter a url every time, just derive it using the version. Not sure why I didn't drop them.